client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -234,36 +272,32 @@ const Checkout = React.createClass( {
 					{ persistent: true }
 				);
 			}
-
-			return purchasePaths.managePurchase( renewalItem.extra.purchaseDomain, renewalItem.extra.purchaseId );
 		} else if ( cartItems.hasFreeTrial( cart ) ) {
 			this.props.clearSitePlans( selectedSiteId );
+		}
 
-			return selectedSiteSlug
-				? `/plans/${ selectedSiteSlug }/thank-you`
-				: '/checkout/thank-you/plans';
-		} else if ( isDomainOnly && cartItems.hasDomainRegistration( cart ) && ! cartItems.hasPlan( cart ) ) {
-			// TODO: Use purchased domain name once it is possible to set it as a primary domain when site is created.
-			return domainManagementList( selectedSite.slug );
+		if ( cart.create_new_blog ) {
+			notices.info(
+				this.translate( 'Almost done…' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Almost done' )` *ES Score: 6.79*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/create-cart-without-site)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).